### DISCUSSION TITLE
WiFi adapter channel capabilities

### DISCUSSION THREAD

**Discussion by afarley** [nzymedefense/nzyme#530] [Questions] [answered]
In looking at the capabilities of my USB  WiFi adapters (Panda PAU09 and Alfa AWUS051NH .v2 and which both use the rt2800usb driver) it looks like the radios max out at channel 104. That would indicate that a substantial number of 5 GHz channels are not available. Is that right? I run my AP's in the higher channels and this might explain why I can't connect certain devices to the network.

Are there adapters with higher frequency 5 GHz radios  that actually work with Nzyme and other monitor mode requiring applications?

**Comment by lennartkoopmann** [comment, marked as answer]
My PAU09 goes up to channel 110. I don't have another one to test right now but I've seen several that go into the 160 area.

Check what country code is enabled on your adapters. Europe and the US should be pretty similar and go high up but Japan stops somewhere in the 140's I believe.

In the end, your driver and hardware combination will dictate this and `iwlist channel` will show it to you in a good overview.
